src/arch/x86/image/multiboot2_entry.S

```diff
@@ -61,8 +61,22 @@ multiboot2_entry:
 	leaq	entry_gdtr(%rip), %rax
 	lgdt	(%rax)
 
-	/* Load our new %cs. */
-	ljmp	*newcs_vector
+	/*
+	 * Load our new %cs.
+	 *
+	 * Note that originally, this was an ljmp; while this worked on many
+	 * systems, it appeared to have issues on at least VirtualBox (see
+	 * https://www.virtualbox.org/ticket/19414), so we set up on iretq frame
```

Comment:
Grammar nit:  "on iretq frame"?  Or "an iretq frame"?

Reply:
Also, did this fix the world in Linux KVM too? Testing info on ticket suggests it might've.

Reply:
done